rundeckapp/grails-spa/packages/ui-trellis/src/library/components/widgets/rundeck-info/RundeckInfo.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { mount } from "@vue/test-utils";
+import RundeckInfo from "./RundeckInfo.vue";
+jest.mock("../../../rundeckService", () => ({
+  getRundeckContext: jest.fn().mockReturnValue({ rdBase: "mocked-rdBase" }),
+  url: jest.fn().mockReturnValue("http://localhost"),
+}));
+const mountRundeckInfo = async (props = {}) => {
+  return mount(RundeckInfo, {
+    props: {
+      appInfo: { title: "Rundeck", logocss: "some-css" },
+      version: {
+        number: "1.0.0",
+        tag: "stable",
+        name: "Erebus",
+        color: "red",
+        icon: "glass",
+        edition: "Community",
+      },
+      server: {
+        name: "localHost",
+        icon: "paperclip",
+        uuid: "uuid1",
+      },
+      latest: {
+        title: "Rundeck",
+        full: "v5.2.0-20240410",
+        number: "5.2.0",
+        tag: "GA",
+        color: "aquamarine",
+        date: new Date(),
+        icon: "knight",
+        edition: "Community",
+      },
+      ...props,
+    },
+  });
+};
+describe("RundeckInfo", () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  it("renders RundeckLogo when appInfo.title is 'Rundeck'", async () => {
+    const wrapper = await mountRundeckInfo();
+    expect(wrapper.findComponent({ name: "RundeckLogo" }).exists()).toBe(true);
+    expect(wrapper.findComponent({ name: "PagerdutyLogo" }).exists()).toBe(
+      false,
+    );
+  });
+  it("renders PagerdutyLogo when appInfo.title is 'Pagerduty'", async () => {
+    const wrapper = await mountRundeckInfo({ appInfo: { title: "Pagerduty" } });
+    expect(wrapper.findComponent({ name: "PagerdutyLogo" }).exists()).toBe(
+      true,
+    );
+    expect(wrapper.findComponent({ name: "RundeckLogo" }).exists()).toBe(false);
+  });
+  it("renders RundeckVersions information with correct props", async () => {
+    const wrapper = await mountRundeckInfo();
+
+    const rundeckVersionComponent = wrapper.findComponent({
+      name: "RundeckVersion",
+    });
+    expect(rundeckVersionComponent.exists()).toBe(true);
+
+    expect(rundeckVersionComponent.find("span").text()).toBe("Rundeck 1.0.0");
+  });
+  it("renders ServerDisplay correctly with default props", async () => {
+    const wrapper = await mountRundeckInfo();
+    const serverComponent = wrapper.findComponent({ name: "ServerDisplay" });
+    expect(serverComponent.exists()).toBe(true);
+
+    expect(
+      serverComponent.find('[data-test-id="server-title"]').attributes("title"),
+    ).toBe("paperclip-uu / uuid1");
```

Comment:
Please add here another expect that checks that there's a span with the text equal to the short uuid